packages/default/scss/gantt/_layout.scss

```diff
@@ -485,11 +485,6 @@
             right: 0;
         }
 
-
-        .k-grid-header .k-header {
-            position: static;
```

Comment:
This code is added here:
https://github.com/telerik/kendo-themes/commit/96e9e1fc8ea359661a2e60c94cff5f414564ed4e

Based on the commit, I am not sure what was the problem, however, this rule breaks the filters/column menus in an RTL Grid